docs/analyzer/user_guide.md

```diff
@@ -1243,9 +1244,18 @@ cross translation unit analysis arguments:
                         analysis, using already available extra files in
                         '<OUTPUT_DIR>/ctu-dir'. (These files will not be
                         cleaned up in this mode.)
-  --ctu-on-the-fly      If specified, the 'collect' phase will not create the
-                        extra AST dumps, but rather analysis will be run with
-                        an in-memory recompilation of the source files.
+  --ctu-ast-creation {from-pch, parsed-on-demand}
+                        Choose the way ASTs are loaded during CTU analysis.
```

Comment:
This help message is different from the help message written in the source code.